content/en/real_user_monitoring/application_monitoring/ios/setup.md

```diff
@@ -98,50 +104,28 @@ DatadogRUM.xcframework
 
    {{< img src="real_user_monitoring/ios/ios-create-application.png" alt="Create a RUM application for iOS in Datadog" style="width:100%;border:none" >}}
 
-[1]: https://app.datadoghq.com/rum/application/create
-[2]: /real_user_monitoring/ios/web_view_tracking/
-[3]: /real_user_monitoring/ios/data_collected/
-
-{{% /tab %}}
-{{% tab "Error Tracking" %}}
-
-1. Navigate to [**Error Tracking** > **Settings** > **Browser and Mobile** > **Add an Application**][1].
-2. Select `iOS` as the application type and enter an application name to generate a unique Datadog application ID and client token.
-3. To instrument your web views, click the **Instrument your webviews** toggle. For more information, see [Web View Tracking][2].
-4. To disable automatic user data collection for either client IP or geolocation data, use the toggles for those settings. For more information, see [iOS Data Collected][3].
-
-   {{< img src="real_user_monitoring/error_tracking/mobile-new-application-1.png" alt="Create an application for iOS in Datadog" style="width:90%;">}}
-
-[1]: https://app.datadoghq.com/error-tracking/settings/setup/client
-[2]: /real_user_monitoring/ios/web_view_tracking/
-[3]: /real_user_monitoring/ios/data_collected/
-
-{{% /tab %}}
-{{< /tabs >}}
-
-To ensure the safety of your data, you must use a client token. If you used only [Datadog API keys][3] to configure the `dd-sdk-ios` library, they would be exposed client-side in the iOS application's byte code.
+To ensure the safety of your data, you must use a client token. Using only [Datadog API keys][2] to configure the `dd-sdk-ios` library would expose them client-side in your iOS application's byte code.
 
-For more information about setting up a client token, see the [Client token documentation][4].
+For more information about setting up a client token, see the [Client token documentation][3].
 
 ### Step 3 - Initialize the library
 
 In the initialization snippet, set an environment name, service name, and client token.
 
-The SDK should be initialized as early as possible in the app lifecycle, specifically in the `AppDelegate`'s `application(_:didFinishLaunchingWithOptions:)` callback. This ensures all measurements, including application startup duration, are captured correctly. For apps built with SwiftUI, you can use `@UIApplicationDelegateAdaptor` to hook into the `AppDelegate`.
+The SDK should be initialized as early as possible in the app lifecycle, specifically in the `AppDelegate`'s `application(_:didFinishLaunchingWithOptions:)` callback. The `AppDelegate` is your app's main entry point that handles app lifecycle events. 
+
+This ensures all measurements, including application startup duration, are captured correctly. For apps built with SwiftUI, you can use `@UIApplicationDelegateAdaptor` to hook into the `AppDelegate`.
```

Comment:
```suggestion
This ensures the SDK can correctly capture all measurements, including application startup duration. For apps built with SwiftUI, you can use `@UIApplicationDelegateAdaptor` to hook into the `AppDelegate`.
```
Not sure if this is right; I was just trying to use active voice. Ignore me if this makes less sense!